File: Math/codeforces_1841A.py

# Problem link ===>> https://codeforces.com/problemset/problem/1841/A

import sys

def main():
    t = int(input())
    for _ in range(t):
        n = int(input())
        if n >= 5:
            print("Alice")
        else:
            print("Bob")

if __name__ == "__main__":
    main()

File: Math/codeforces_1841A.cpp

// Problem link ===>> https://codeforces.com/problemset/problem/1841/A
// submission link ===>> https://codeforces.com/contest/1841/submission/209977694

#include <iostream>
using namespace std;

int main()
{
    int t;
    cin >> t;
    while (t--)
    {
        int n;
        cin >> n;
        if (n >= 5)
            cout
                << "Alice\n";
        else
            cout << "Bob\n";
    }
    return 0;
}
